src/dlgPackageExporter.cpp

```diff
@@ -215,7 +254,17 @@ void dlgPackageExporter::slot_export_package()
         }
     }
 
-    writer.exportPackage(filePath);
+    if (!writer.exportPackage(mXmlPathFileName)) {
+        displayResultMessage(tr("Failed to export - could not write Mudlet items to %1.").arg(mXmlPathFileName),
+                             false);
+        // Although we have failed we must not just abort here we need to reset
+        // the selected for export or not flags first - so note that we have
+        // failed:
+        isOk = false;
```

Comment:
what is ok? Maybe mention export or writable

Reply:
The whole method/process/procedure has worked *so far* - this is used as an error flag to skip further processing if things are NOT okay - it needs to be that way because the code is not structured in a way that `return`-ing early in the event of an error will work.

Reply:
Understanding the need for this variable. Commenting on its name though. 
"exportFailed = true" seems much easier to grasp later then "isOk" = false"

Reply:
It might not be the *export* (the writing of the final file itself) that failed - saying *exportFailed* suggests to me that the flag is specific to that operation (though I agree that the whole method is aimed at achieving that goal).

As it happens `isOk` is a flag that I have seen used in the Qt examples - and it is nice and short but still meaningful - and I *have* used it elsewhere - in ***22*** places when I just checked...! :wink: